src/clr-angular/forms/common/providers/control-class.service.spec.ts

```diff
@@ -55,7 +55,7 @@ export default function(): void {
 
     it('should return any additional classes passed by the control', function() {
       service.className = 'test-class';
-      expect(service.controlClass(false, false, 'extra-class')).toBe('test-class extra-class');
+      expect(service.controlClass(false, false, 2, 'extra-class')).toBe('test-class extra-class');
```

Comment:
Please add a test that confirms that if you pass a different labelSize that it renders the right class.